### PR TITLE
Organized notebook file, cleaned up directories, included example with tumor image

### PR DESCRIPTION
**Pull request by colinrec34**
Cleaned up a lot of the file structuring, tried to make it as easy as possible to import a laser image/dcm image for a clear and concise way to run the simulation. The example included is for 10000 particles travelled 0.25 meters from the cathode. Switched the plot saving structure to .jpg so the full plot is saved rather than just the points of the plot. Also added comparison plots of the shape of the beam before and after the simulation.

### CODE REVIEW

File: astra/astra-inputs/distgen.yaml [hunk 1]
```diff
@@ -11,19 +11,31 @@ total_charge:
...
+
 t_dist:
   type: gaussian
   avg_t:
     units: ps
     value: 0
   sigma_t:
     units: ps
-    value: 0.85
+    value: 32.87
```
Comment (nneveu) [inline astra/astra-inputs/distgen.yaml:39]: Was this the number we calculated? I thought we calculated 32 ps? This should be the 'sigma' value, not the FWHM, so a fraction of 32? It's not a huge problem for your paper/work/test case, but we should fix it before merging.

File: astra/astra-inputs/distgen.yaml [hunk 2]
```diff
@@ -11,19 +11,31 @@ total_charge:
...
-    value: 0.5
-  
+  max_x:
+    value: 15
+    units: mm
+  min_y: 
+    value: -15
+    units: mm
+  max_y:
+    value: 15
```
Comment (nneveu) [inline astra/astra-inputs/distgen.yaml:27]: Is the beam size 15 mm or the whole picture size?

File: high_energy_opt/highe_opt.yaml [hunk 3]
```diff
@@ -1,73 +0,0 @@
-xopt:
```
Comment (nneveu) [inline high_energy_opt/highe_opt.yaml:1]: Why was this deleted? It's an old optimization, but can go in a different folder?

Comment (nneveu) [reply high_energy_opt/highe_opt.yaml:1]: Same question for some of the older yaml files / dmd images?

**Review by nneveu — CHANGES_REQUESTED** [review]
I haven't looked at the notebooks yet, but I left some small comments on the other files.

**Comment by colinrec34** [comment]
submitting refactored version in separate pull request

**Pull request closed without merging**